src/test/java/com/vmware/vip/i18n/NumberFormatUtilOfflineTest.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vip.i18n;
+
+import com.vmware.vipclient.i18n.I18nFactory;
+import com.vmware.vipclient.i18n.VIPCfg;
+import com.vmware.vipclient.i18n.base.cache.FormattingCache;
+import com.vmware.vipclient.i18n.base.instances.NumberFormatting;
+import com.vmware.vipclient.i18n.exceptions.VIPClientInitException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Locale;
+
+public class NumberFormatUtilOfflineTest extends BaseTestClass {
+
+    NumberFormatting numberFormatI18n;
+
+    @Before
+    public void init() {
+        VIPCfg gc = VIPCfg.getInstance();
+        try {
+            gc.initialize("vipconfig-offline");
```

Comment:
Same comment: "This will cause other tests to fail because VIPCFG is a singleton. I had added vipconfig-offline before, but Colin and Xiaochao do not want to use it inside the tests because they say that we should not be able to re-initialize VIPCfg. Hence, that .properties file is not used anymore. Only vipconfig.properties is used, and if I have to modify any property for certain unit tests, I do it inside the test itself. See LocaleTest.testGetSupportedLocalesOfflineBundles for an example. "

Reply:
Same resolution as above.